packages/services/src/models/notification-scanners/SubscriptionReminder.js

```diff
@@ -0,0 +1,34 @@
+import BaseModel from './BaseModel'
+import { User } from '@aragonone/court-backend-server/build/models/objection'
+
+class SubscriptionReminder extends BaseModel {
+  async scan() {
+    let notifications = []
+    const users = await User.query().where({emailVerified: false}).withGraphFetched('[email,emailVerificationToken]')
+    for (const user of users) {
+      /**
+       * Generate subscription reminders in two cases:
+       * 1) If user has an existing verification token that expired a day ago
+       * 2) If user has no token and address was never verified (previous subscriptions from https://anj.aragon.org/)
+       */
+      if (!user.email) continue
+      if (!user.emailVerificationToken && user.addressVerified) continue
+      if (user.emailVerificationToken && user.emailVerificationToken.expiresAt > new Date(Date.now()-this._DAYS)) continue
```

Comment:
Maybe I'm confused, but haven't we been repeating this logic in other places? Is it something we could abstract in the user under a method like `user.hasSubscribed()`

Reply:
Yeah I just realised if I have to do comments it's probably too complicated :)
Logic is not repeated in other places this is specifically to find expired tokens or anj registrations.
I have abstracted a bit, should be more clear now, let me know.